Source/ULibs.Win32/RELEASENOTES.md

```diff
@@ -0,0 +1,7 @@
+﻿# ULibs.Win32 release notes
+
+## 0.1.0
+
+### Features
+
+- Describe the first feature here.
```

Comment:
This is also boilerplate and needs to be updated. Also, the package version number is derived from the first entry in this file, so you might want to consider changing it. Libraries are expected to use semantic versioning.